bundle/manifests/update-service-operator.clusterserviceversion.yaml

```diff
@@ -35,14 +35,6 @@ spec:
       displayName: Update Service
       kind: UpdateService
       name: updateservices.updateservice.operator.openshift.io
-      resources:
```

Comment:
https://github.com/openshift/cincinnati-operator/pull/131/files 

/hold

Let me figure out this one.